.travis.yml

```diff
@@ -14,4 +13,3 @@ before_script:
 addons:
   hosts:
     - dynamodb-local
-script: "MIX_ENV=test mix local.hex --force && MIX_ENV=test mix do deps.get, test"
```

Comment:
Hmm, by right since this is the default command you can actually omit it, but I wonder why your tests are breaking.